packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -278,38 +278,28 @@ const prepareY2Axis = (scale, tickValues, height, format) => d3
  */
 const getYAndY2AxisFormat = (tickValues, axisData) => {
   let tickDecimalMagnitude = 0;
-  let largestDecimalValue = 0;
   tickValues.forEach((tick) => {
     if (tick !== 0) {
       let tempTick = tick;
       tempTick = Math.abs(tick);
       const tempTickDecimalMagnitude = -Math.floor(Math.log10(tempTick));
       if (tempTickDecimalMagnitude > tickDecimalMagnitude) {
         tickDecimalMagnitude = tempTickDecimalMagnitude;
-        largestDecimalValue = tick;
       }
     }
   });
 
-  const setFormat = (lowerRangeValue, upperRangeValue) => {
-    if (largestDecimalValue > lowerRangeValue && largestDecimalValue < upperRangeValue && largestDecimalValue !== 0) {
-      let format;
-      if (axisData.suppressTrailingZeros) {
-        format = `.${tickDecimalMagnitude % 10}~f`;
-      } else {
-        format = `.${tickDecimalMagnitude % 10}f`;
-      }
-      return format;
+  let format;
+  if (tickDecimalMagnitude > 0 && tickDecimalMagnitude < 6) {
+    if (axisData.suppressTrailingZeros) {
+      format = `.${tickDecimalMagnitude % 10}~f`;
+    } else {
+      format = `.${tickDecimalMagnitude % 10}f`;
     }
-    return undefined;
-  };
-
-  if (tickDecimalMagnitude > 5) {
-    return undefined;
+    return format;
   }
-  const rangeValue = 5 / (10 ** tickDecimalMagnitude);
 
-  return setFormat(-rangeValue, rangeValue);
+  return undefined;
```

Comment:
nit: Does it work the same if you just do this?

```
  let format;
  if (tickDecimalMagnitude > 0 && tickDecimalMagnitude < 6) {
    if (axisData.suppressTrailingZeros) {
      format = `.${tickDecimalMagnitude % 10}~f`;
    } else {
      format = `.${tickDecimalMagnitude % 10}f`;
    }
  }

  return format;
```

Reply:
It won't work that way Ben because we have to return undefined for sure if our tick magnitude is out of range(beyond 5).

Reply:
Does `let format` not default format to undefined?

Reply:
Yes you are right Ben. Fixed [here](https://github.com/cerner/terra-graphs/pull/267/commits/2f2cc11e5a7b8d72b10a9fa3ed29a3199392c57d#).